server/events/output_updater.go

```diff
@@ -116,45 +96,28 @@ func (c *ChecksOutputUpdater) UpdateOutput(ctx *command.Context, cmd PullCommand
 }
 
 func (c *ChecksOutputUpdater) handleApprovePolicies(ctx *command.Context, cmd PullCommand, res command.Result) {
-	output := c.MarkdownRenderer.Render(res, cmd.CommandName(), ctx.Pull.BaseRepo)
```

Comment:
The Command Runner handles this: 
https://github.com/lyft/atlantis/blob/8cf99abfd260c9da2ab0db5cf9aa940e896de4be/server/events/approve_policies_command_runner.go#L77